templates/middleware.js

```diff
@@ -56,25 +56,38 @@ export const APP_MIDDLEWARE = [
 ]
 // {% else %}
 module.exports = {
-  // You can export middleware for testing or for
-  // attaching to routes.
+  // You can export middleware for testing or for attaching to routes.
   routeMiddlewareFunc,
   setupMiddlewareFunc,
   APP_MIDDLEWARE: [
-    // This export is special: it instructs Boltzmann to attach
-    // middlewares to the app in this order.
-    // This is also where you can configure built-in middleware.
+    // This export is special: it instructs Boltzmann to attach middlewares to the app in this
+    // order. This is also where you can configure built-in middleware.
     setupMiddlewareFunc,
+    // Must be one of DENY or SAMEORIGIN; uncomment to enable
```

Comment:
maybe wrap in a `{% if website %}`?

Reply:
yeah, that's kind of a function of the middleware not bringing in other deps. I was thinking (maybe mistakenly?) that it wouldn't make much sense for an API server to send XFO headers, so we should turn them off unless the user explictly said that they're building a website.